Doc/Tutorial/chapter_entrez.tex

```diff
@@ -200,18 +200,19 @@ \section{ESearch: Searching the Entrez databases}
 
 As a final example, let's get a list of computational journal titles:
 \begin{verbatim}
->>> handle = Entrez.esearch(db="journals", term="computational")
+>>> handle = Entrez.esearch(db="nlmcatalog", term="computational[Journal]", , RetMax='20')
```

Comment:
Also it should be ``retmax`` not ``RetMax`` although it "works" and the case is carried forward to the URL.